packaging/package.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.Build.NoTargets/3.5.6">
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+        <IsPackable>true</IsPackable>
+        <NoWarn>NU5100;NU5128</NoWarn>
+
+        <!-- MSBuild tasks shouldn't be referenced. This is by design. -->
+        <NoWarn>NU5100;NU5128</NoWarn>
+        <Authors>Microsoft</Authors>
+        <Company>Microsoft</Company>
+        <Packageid>Microsoft.NET.Build.Containers</Packageid>
+        <Description>Tasks and targets to natively publish .NET applications as containers.</Description>
+        <Copyright>&#169; Microsoft Corporation. All rights reserved.</Copyright>
+        <PackageLicenseExpression>MIT</PackageLicenseExpression>
+        <PackageProjectUrl>https://github.com/dotnet/sdk-container-builds</PackageProjectUrl>
+        <RepositoryUrl>https://github.com/dotnet/sdk-container-builds</RepositoryUrl>
+        <RepositoryType>git</RepositoryType>
+        <PackageTags>containers;docker;Microsoft.NET.Build.Containers</PackageTags>
+        <PackageReadmeFile>README.md</PackageReadmeFile>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="../containerize/containerize.csproj" ReferenceOutputAssembly="true" />
+        <ProjectReference Include="../Microsoft.NET.Build.Containers/Microsoft.NET.Build.Containers.csproj" OutputItemType="ContainerLibOutput"/>
+        <PackageReference Include="System.CommandLine" IncludeAssets="runtime"/>
+    </ItemGroup>
+
+
+    <Target Name="DoThePackaging" AfterTargets="Build">
+        <ItemGroup>
+            <Content Include="$(OutDir)containerize.dll" Pack="true" PackagePath="containerize\" />
+            <Content Include="$(OutDir)containerize.exe" Pack="true" PackagePath="containerize\" />
+            <Content Include="$(OutDir)containerize.runtimeconfig.json" Pack="true" PackagePath="containerize\" />
+            <Content Include="$(OutDir)System.CommandLine.dll" Pack="true" PackagePath="containerize\" />
+            <Content Include="@(ContainerLibOutput)" Pack="true" PackagePath="containerize\" />
```

Comment:
This bit feels like what you're asking for is 'gimme the runtime assets of my ProjectReference and put them _here_'. Is there an easier way to ask for the runtime assets of a ProjectReference? I'm guessing no, but it would be awesome if there was.

Reply:
> Is there an easier way to ask for the runtime assets of a ProjectReference?

I'm not aware of one, but I'm open to learning about / creating a "canonical" way to do this.
> Maybe something with GetCopyToOutputDirectoryItems or GetCopyToPublishDirectoryItems?

To be clear, you're suggesting an MSBuild task call on the project reference, calling one of these targets and gathering the outputs?

Reply:
I was hoping we could set the `Targets` metadata on the `ProjectReference` and put it in a new item.